estimagic/dashboard/monitoring_app.py

```diff
@@ -171,28 +192,37 @@ def _calculate_start_point(database, rollover, jump):
 
 
 def _create_initial_convergence_plots(
-    criterion_history, params_history, group_to_params
+    criterion_history, params_history, group_to_param_ids, start_params,
 ):
     """Create the initial convergence plots.
 
     Args:
         criterion_history (bokeh ColumnDataSource)
         params_history (bokeh ColumnDataSource)
-        group_to_params (dict):
+        group_to_param_ids (dict):
             Keys are the values of the "group" column. The values are lists with
             bokeh friendly strings of the index tuples identifying the parameters
             that belong to this group. Parameters where group is None, "" or False
             are ignored.
+        start_params (pd.DataFrame): See :ref:`params`. It includes a dashboard
+            specific id column.
 
     Returns:
         convergence_plots (list): List of bokeh Row elements, each containing one
             convergence plot.
 
     """
     param_plots = []
-    for g, group_params in group_to_params.items():
+    for group, param_ids in group_to_param_ids.items():
+        param_names = [
+            start_params.query(f"id == '{id_}'")["name"].iloc[0] for id_ in param_ids
```

Comment:
If we abstract the `group_to_params_ids` function a bit, such that it can map groups to any column we could re-use to get a mapping from groups to names. This would simplify the code here.

Reply:
Agreed, much nicer this way. (0081c97)